app.py

```diff
@@ -27,7 +27,8 @@
         'morning_checks': Cron('0', '10', '*', '*', '?', '*'),
         'morning_checks_2': Cron('15', '10', '*', '*', '?', '*'),
         'monday_checks': Cron('0', '9', '?', '*', '2', '*'),
-        'monthly_checks': Cron('0', '9', '1', '*', '?', '*')
+        'monthly_checks': Cron('0', '9', '1', '*', '?', '*'),
+        'manual_checks': Cron('0', '0', '31', '2', '?', '?')  # every February 31st aka never
```

Comment:
Uhhh. Wow. In general I wish for these to be expressed in a more readable way, though we don't have to solve that today. But as to this one new need, how about...
```
def effectively_never():
    """Every February 31st, a.k.a. 'never'."""
    return Cron('0', '0', '31', '2', '?', '?')
```

Reply:
Can do. Probably a good idea anyway in case some mechanism is introduced to detect such schedules.